ide-common/src/main/kotlin/org/digma/intellij/plugin/posthog/ActivityMonitor.kt

```diff
@@ -17,6 +17,18 @@ import java.time.LocalDateTime
 
 class ActivityMonitor(private val project: Project) /*: Runnable, Disposable*/ {
 
+    enum class Panel{
```

Comment:
too generic name lets make it something of our own
```suggestion
    enum class MonitoredPanel
```

Reply:
renamed